src/git/zcl_abapgit_git_porcelain.clas.abap

```diff
@@ -216,7 +216,7 @@ CLASS zcl_abapgit_git_porcelain IMPLEMENTATION.
       CLEAR lt_nodes.
 
 * files
-      LOOP AT it_expanded ASSIGNING <ls_exp> USING KEY path WHERE path = <ls_folder>-path.
+      LOOP AT it_expanded ASSIGNING <ls_exp> USING KEY path_name WHERE path = <ls_folder>-path.
```

Comment:
I don't partial keys in where clause work in 702 (which was the reason for adding another key). Let me double check

Reply:
it works fine. merging